Src/Public/Reports/K2/k2.ps1

```diff
@@ -0,0 +1,121 @@
+#requires -Module @{ModuleName="PScribo";ModuleVersion="0.7.23"},@{ModuleName="K2.Powershell";ModuleVersion="1.4.5.2"}
+
+<#
+.SYNOPSIS  
+    PowerShell script to document the configuration of Kaminario Array in Word/HTML/XML/Text formats
+.DESCRIPTION
+    Documents the configuration of Kaminario Array in Word/HTML/XML/Text formats using PScribo.
+.NOTES
+    Version:        0.1
+    Author:         Nick Lepore
+    Twitter:        @midnigh7
+    Github:         Midnigh7
+
+    Credits:        Iain Brighton (@iainbrighton) - PScribo module
+                    Tim Carman (@tpcarman)
+
+.LINK
+    https://github.com/tpcarman/As-Built-Report
+    https://github.com/iainbrighton/PScribo
+    https://www.powershellgallery.com/packages/K2.Powershell/
+#>
+
+#region Configuration Settings
+###############################################################################################
+#                                    CONFIG SETTINGS                                          #
+###############################################################################################
+
+# If custom style not set, use Kaminario style
+if (!$StyleName) {
+    .\Styles\K2Storage.ps1
+}
+
+# Connect to Kaminario Arrays using supplied credentials
+$K2Arrays = $Target.split(",")
+foreach ($Endpoint in $K2Arrays) {
+    $K2User = ($Credentials).Username
+    $K2Password = ($Credentials.GetNetworkCredential()).password
+    Connect-K2Array -K2Array $Endpoint -Username $K2User -Password $K2Password
+
+    #endregion Configuration Settings
+
+    #region Script Body
+    ###############################################################################################
+    #                                       SCRIPT BODY                                           #
+    ###############################################################################################
+
+    $ArrayName = (Get-K2State).system_name
+    Section -Style Heading1 $Arrayname {
+        Section -Style Heading2 'System Summary' {
+            Section -Style Heading3 'Array Summary' {
+                $ArraySummary = Get-K2State | Sort-Object array_name | Select-Object @{L = 'System Name'; E = {$_.system_name}}, @{L = 'Version'; E = {$_.system_version}}, 

```

Comment:
We are working to improve readability and consistency in existing scripts, so we are moving to using PSCustomObjects, for eaxample;
``$Array = Get-K2State | Sort-Object array_name
$ArraySummary = [PSCustomObject] @ {
    'System Name' = $Array.system_name
    'Version' = $Array.system_version
    'Connectivity Type' = $Array.system_connectivity_type
    'System State' = $Array.state
}
$ArraySummary | Table -Name 'Array Summary'``

Could you please update your script to follow these new guidelines.